NuGet.config

```diff
@@ -2,20 +2,12 @@
 <configuration>
   <!-- Don't use any higher level config files.
        Our builds need to be isolated from user/machine state -->
-  <fallbackPackageFolders>
-    <clear />
-  </fallbackPackageFolders>
```

Comment:
❗  We definitely want to keep this section. Windows Desktop maintains its own packages folder under .\.packages

Reply:
How is this used? this is an input to build if i understand and the section is empty here.

Reply:
https://docs.microsoft.com/nuget/reference/nuget-config-file#fallbackpackagefolders-section
It will cause a build to fail if it can't find a reference locally, and prevent it from searching for a reference elsewhere.